apps/climatemappedafrica/src/components/ExplorePage/index.js

```diff
@@ -5,17 +5,16 @@ import PropTypes from "prop-types";
 import React, { useEffect, useState } from "react";
 
 import useExplore from "./useExplore";
+import { primaryGeoStyles, secondaryGeoStyles } from "./useLayerStyles";
 import useProfileGeography from "./useProfileGeography";
 import useStyles from "./useStyles";
 
 import Location from "@/climatemappedafrica/components/HURUmap/Location";
 import Panel from "@/climatemappedafrica/components/HURUmap/Panel";
 
-const Map = dynamic(
-  () => import("@/climatemappedafrica/components/HURUmap/Map"),
-  {
-    ssr: false,
-  },
+const DynamicMap = dynamic(
```

Comment:
```suggestion
const Map = dynamic(
```

Not sure why it **needs** to be named Dynamic.